azurerm/resource_arm_availability_set.go

```diff
@@ -90,13 +99,24 @@ func resourceArmAvailabilitySetCreateUpdate(d *schema.ResourceData, meta interfa
 	managed := d.Get("managed").(bool)
 	tags := d.Get("tags").(map[string]interface{})
 
+	properties := compute.AvailabilitySetProperties{
+		PlatformFaultDomainCount:  utils.Int32(int32(faultDomainCount)),
+		PlatformUpdateDomainCount: utils.Int32(int32(updateDomainCount)),
+	}
+
+	if v, ok := d.GetOk("proximity_placement_group_id"); ok {
```

Comment:
I have moved this and I have since moved it back because in short no not really. This is related to your other comment about availability properties. I just remembered why I changed it to this. I broke out availability set properties because the ppg is an optional property so I used the same pattern I saw elsewhere to add optional properties.